src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -106,6 +104,8 @@
 
   private static List<TranscriptLine> shortVideoTranscriptLines;
   private static List<TranscriptLine> longVideoTranscriptLines;
+  Key lectureKeyA;
+  Key lectureKeyB;
```

Comment:
Because `parseAndStoreTranscript` takes in a key rather than a long, `TranscriptUtil` has to be updated along with the method `putTranscriptLinesInDatastore` that uses it.

Reply:
Can these be private static?

Reply:
Thanks for catching that! done